src/Zaaksysteem/Foundation/Helpers.php

```diff
@@ -100,3 +102,35 @@ function get_supplier(array $form, bool $getKey = false): string
 
     return $allowed[$supplier] ?? '';
 }
+
+/**
+ * Add form field values to arguments required for creating a 'Zaak'.
+ * Mapping is done by the relation between arguments keys and form fields linkedFieldValueZGWs.
+ */
+function field_mapping(array $fields, array $entry): array
+{
+    $mappedFields = [];
+
+    foreach ($fields as $field) {
+        if (empty($field->linkedFieldValueZGW)) {
+            continue;
+        }
+
+        $property = \rgar($entry, (string)$field->id);
+
+        if (empty($property)) {
+            continue;
+        }
+
+        if ($field->type === 'date') {
+            $property = (new DateTime($property))->format('Y-m-d');
```

Comment:
Wees je er van bewust dat `DateTime` een `Exception` kan genereren wanneer `$property` geen valide datum/tijd is.

Reply:
Misschien dat dit stuk eruit kan, ik heb niet zo gek veel veranderd aan deze functie maar tot dusver ben ik ook nog geen datum velden tegengekomen welke door een gebruiker ingevuld moeten worden.